packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -1052,16 +1052,16 @@ abstract class ResidentRunner {
       commandHelp.P.print();
       commandHelp.a.print();
     }
-    if (flutterDevices.any((FlutterDevice d) => d.device.supportsScreenshot)) {
-      commandHelp.s.print();
-    }
   }
 
   /// Called when a signal has requested we exit.
   Future<void> cleanupAfterSignal();
 
   /// Called right before we exit.
   Future<void> cleanupAtFinish();
+
+  // Clears the screen.
+  void clearScreen([Logger logger]) => (logger ?? globals.logger).clear();
```

Comment:
Rather than using an optional logger, you could configure the test logger via the test context instead. See below:

Reply:
During regular runtime (`lib/src/context_runner.dart`), the logger is configured like this: 
```dart
Logger: () => globals.platform.isWindows
  ? WindowsStdoutLogger(
      terminal: globals.terminal,
      stdio: globals.stdio,
      outputPreferences: outputPreferences,
      timeoutConfiguration: timeoutConfiguration,
      platform: globals.platform,
   )
  : StdoutLogger(
      terminal: globals.terminal,
      stdio: globals.stdio,
      outputPreferences: outputPreferences,
      timeoutConfiguration: timeoutConfiguration,
      platform: globals.platform,
   ),
```

During testing (`test/src/context.dart`), the logger is configured like this: 
```dart
Logger: () => BufferLogger(
  terminal: globals.terminal,
  outputPreferences: outputPreferences,
),
```

So I'm going to assume this means I can always call `globals.logger` to get the right one, but correct me if I'm wrong.